packages/core/content-manager/server/controllers/relations.js

```diff
@@ -1,64 +1,96 @@
 'use strict';
 
-const { prop, pick } = require('lodash/fp');
+const { prop, isEmpty, isNil } = require('lodash/fp');
+const { convertFiltersQueryParams } = require('@strapi/utils/lib/convert-query-params');
+const { hasDraftAndPublish } = require('@strapi/utils').contentTypes;
 const { PUBLISHED_AT_ATTRIBUTE } = require('@strapi/utils').contentTypes.constants;
 
 const { getService } = require('../utils');
+const { validateFindNew } = require('./validation/relations');
 
 module.exports = {
-  async find(ctx) {
+  async findNew(ctx) {
```

Comment:
Not sure about the naming 🤔 It is finding available/unrelated entities? new is putting me off

Reply:
`findAvailable` works for me too. Yes it's to find the new relations that we can add to an entity